network/robus_network/src/reception.c

```diff
@@ -243,7 +243,7 @@ _CRITICAL void Recep_GetCollision(luos_phy_t *phy_robus, volatile uint8_t *data)
             // Collision detection end
             data_count = 0;
             RobusHAL_SetRxState(false);
-            RobusHAL_ResetTimeout(0);
+            RobusHAL_ResetTimeout(false, 0);
```

Comment:
Here, you just want to shut down the timer so putting the value to 0 makes sense. You could remove the boolean value and just check if the value is 0.